share/goodie/ia_hints/title.handlebars

```diff
@@ -0,0 +1 @@
+<span>Pssst! See what happens when you search for </span><strong><a href="https://duckduckgo.com/?q={{sample_query}}">"{{sample_query}}"</a></strong>
```

Comment:
I tried this with a relative URL - I actually think for demo purposes it works better with an absolute path. If you try running this with `duckpan server`, having a relative URL will redirect you to a page that says "Sorry, no hit for your Instant Answer".

(I tried this locally but I think this should still be the case with Codio - lemme know if I'm mistaken here.)

But yeah, this should be relative if this was to go live.